tests/test_archive_querier.py

```diff
@@ -231,15 +231,19 @@ def test_deduplicating_work_id_records(table_maker, querier):
     assert len(results) == 1
 
 
-def get_multiple_pages(query_function, query_args):
+def get_multiple_pages(query_function, query_args, allow_single_page=False):
```

Comment:
Can we drop the boolean allow_single_page=True argument? boolean arguments like this are usually a signal that you should really have two functions. And you don't seem to use the new argument anyway.